Control.cpp

```diff
@@ -0,0 +1,492 @@
+#include "Control.h"
+#include <NewPing.h>
+
+#define SONAR_NUM 6      // Number of sensors.
+#define MAX_DISTANCE 200 // Maximum distance (in cm) to ping.
+
+NewPing sonar[SONAR_NUM] = {   // Sensor object array.
+  NewPing(4, 5, MAX_DISTANCE), // Each sensor's trigger pin, echo pin, and max distance to ping. 
+  NewPing(6, 7, MAX_DISTANCE), 
+  NewPing(8, 9, MAX_DISTANCE), 
+  NewPing(4, 5, MAX_DISTANCE), 
+  NewPing(6, 7, MAX_DISTANCE), 
+  NewPing(6, 7, MAX_DISTANCE)
+};
+
+LiquidCrystal_I2C lcd(0x27,20,4);
+
+Control::Control(){
+  bno = Adafruit_BNO055();
+}
+
+void Control::LCDCalibration() {
+  lcd.init(); 
+  lcd.backlight();
+  lcd.print("Hola Mundo");
+  lcd.setCursor(0, 1);
+  lcd.print(millis()/1000);
+  lcd.print(" Segundos");
+  delay(kTimeToPrint);
+}
+
+void Control::BNOCalibration() {
```

Comment:
In general, this and the rest of the functions regarding the use of the BNO, try to move them out to their own file. Like: Bno.cpp and Bno.h, same with the motor movements: Motor.cpp, Motor.h, and import both of them in here. This is to make it more modular and easier to read/debug.